blocks/results-list-block/features/results-list/default.jsx

```diff
@@ -1,407 +1,30 @@
-                      resizedImageOptions={imageURL !== null
-                        ? extractResizedParams(element) : placeholderResizedImageOptions}
-                    />
-                  </a>
-                </div>
-              ) : null}
-              { promoElements.showHeadline ? (
-                <div className="results-list--headline-container mobile-order-1">
-                  <a
-                    href={url}
-                    title={headlineText}
-                  >
-                    <Heading className="headline-text">
-                      {headlineText}
-                    </Heading>
-                  </a>
-                </div>
-              ) : null}
-              { (
-                promoElements.showDescription
-            || promoElements.showDate
-            || promoElements.showByline
-              ) ? (
-                <div className="results-list--description-author-container mobile-order-3">
-                  {promoElements.showDescription && descriptionText && (
-                  <a
-                    href={url}
-                    title={headlineText}
-                  >
-                    <SecondaryFont
-                      as="p"
-                      className="description-text"
-                    >
-                      {descriptionText}
-                    </SecondaryFont>
-                  </a>
-                  )}
-                  { (promoElements.showDate || promoElements.showByline) && (
-                  <div className="results-list--author-date">
-                    { promoElements.showByline
-                    && <Byline content={element} list separator={promoElements.showDate} font="Primary" /> }
-                    { promoElements.showDate && <ArticleDate classNames="story-date" date={displayDate} /> }
-                  </div>
-                  )}
-                </div>
-                ) : null}
-            </div>
-          );
-        })}
-        {
-    !!(contentElements && contentElements.length > 0 && seeMore) && (
-      <div className="see-more">
-        <ReadMoreButton
-          type="button"
-          onClick={() => this.fetchStories(true)}
-          className="btn btn-sm"
-          primaryFont={getThemeStyle(arcSite)['primary-font-family']}
-          primaryColor={getThemeStyle(arcSite)['primary-color']}
-          aria-label={this.phrases.t('results-list-block.see-more-button-aria-label')}
-        >
-          {this.phrases.t('results-list-block.see-more-button')}
-        </ReadMoreButton>
-      </div>
-    )
-  }
-      </div>
-    );
-
-    return (
-      <LazyLoad enabled={this.lazyLoad && !this.isAdmin}>
-        <HeadingSection>
-          <Results />
-        </HeadingSection>
-      </LazyLoad>
-    );
-  }
-}
+import Results from './results';
+
+const ResultsList = () => {
+  const { customFields, isAdmin } = useFusionContext();
```

Comment:
`customFields` is a prop of features in Fusion. Getting them from a fusion hook seems like an anti pattern, and also something that is not consistent with other blocks

Reply:
I thought it was more dirty to grab different values from various places, but I'll change it back.  Makes testing easier...